quicksettings/QuickSettings.hx

```diff
@@ -147,35 +141,57 @@ extern class QuickSettings
 	**/
 	public function addHTML(title:String, htmlString:String):QuickSettings;
 
-	@:deprecated public function getInfo(title:String):String; // use getHTML instead
-	public function getTime(title:String):String;
-	public function getNumber<T:Float>(title:String):T;
-	public function getRangeValue<T:Float>(title:String):T;
-	public function getBoolean(title:String):Bool;
-	public function getColor(title:String):String;
-	public function getText(title:String):String;
-	public function getDate(title:String):String;
-	public function getHTML(title:String):String;
-	public function getFile(title:String):String;
-	public function getDropDownValue<T>(title:String):T;
-	public function getProgressValue<T:Float>(title:String):T;
-	public function getNumberValue<T:Float>(title:String):T;
-	public function getValuesAsJSON(asString:String):String;
-
-	@:deprecated public function setInfo(title:String, text:String):QuickSettings; // use getHTML instead
-	public function setTime(title:String, time:String):QuickSettings;
-	public function setDate(title:String, date:String):QuickSettings;
-	public function setRangeValue(title:String, value:Float):QuickSettings;
-	public function setRangeParameters<T:Float>(title:String, min:T, max:T, step:T):QuickSettings;
-	public function setNumberParameters<T:Float>(title:String, min:T, max:T, step:T):QuickSettings;
-	public function setBoolean(title:String, value:Bool):QuickSettings;
-	public function setColor(title:String, color:String):QuickSettings;
-	public function setText(title:String, text:String):QuickSettings;
-	public function setDropDownIndex(title:String, index:Int):QuickSettings;
-	public function setImageURL(title:String, imageURL:String):QuickSettings;
-	public function setProgressValue<T:Float>(title:String, value:T):QuickSettings;
-	public function setNumberValue<T:Float>(title:String, value:T):QuickSettings;
-	public function setHTML(title:String, html:String):Void;
+	/**
+        Sets up the panel to save all of its values to local storage. This will also immediately try to read in any saved values from local storage, if they exist.
+        So the method should be called after all controls are created on the panel.
+	**/
+	public function saveInLocalStorage (name:String):QuickSettings;
+
+	/**
+        Clears any saved values in local storage.
+    **/
+	public function clearLocalStorage (name:String):QuickSettings;
+
+
+	/**
+		This is the new getter/setter for QuickSettings (version 3). 
```

Comment:
Can you replace the comments to: "Sets value of the control with the given title."

Btw according the original source code it still returns `QuickSettings`